css/style.css

```diff
@@ -1,7 +1,285 @@
+html {
+	box-sizing: border-box; /*padding и border включаются в общую ширину и высоту элемента*/
+}
+
+*, *::before, *::after {
+	box-sizing: inherit;
+}
+
+body {
+	height: 100vh; /*значение высоты опр-ся по viewport*/
+	margin: 0 auto; /*выравнивание по центру*/
+	background: url(../img/iswanto-arif-448003-unsplash.png) no-repeat center; /*фоновая картинка*/
+	background-size: cover; /*картинка полностью покрывает блок при любых размерах*/
+	color: #E8EEE4; /*цвет шрифта*/
+	font: normal 22px/26px MuseoSansCyrl; /*стиль шрифта*/
+	overflow: scroll;
+}
+
+/*header styles start*/
+.container {
+	max-width: 1650px; 
+	max-height: 180px;
+	margin: 0 auto; /*выравнивание по центру*/
+}
+
+.header__row { 
+	display: flex; 
+	align-items: flex-end; /*вертикальное выравнивание по нижнему краю контейнера*/
+	justify-content: space-between; /*выравнивание по ширине - по краям конейнера*/
+	width: 100%;
+	margin-bottom: 9.5%;
+}
+
+.header__logo {
+	padding-top: 6%;
+	width: 146px;
+	margin-left: 10%;
+}
+
+.header__logo img {
+	max-width: 100%;
+	max-height: 100%;
+}
+
+.header__menu {
+	display: flex;
+	justify-content: space-around;
+	margin-right: 6%;
+}
+
+.menu__link {
+	flex-wrap: nowrap;
+	color: #E8EEE4;
+	margin-right: 33.75px; /*отступ справа у пуктов меню*/
+	text-decoration: none; /*убрали нижнее подчеркивание*/
+}
+
+.menu__link:hover, .menu__link:focus, .menu__link:active { /*стили для наведения на ссылки меню*/
+	opacity: .50;
+}
+
+.header__burger { /*бургер-меню*/
+	display: none;
+	width: 25px;
+	height: 17px;
+	position: relative;
+}
+
+.burger__line {
+	display: block;
+	width: 100%;
+	height: 3px;
+	background: #FCF5ED;
+	position: absolute;
+	left: 0;
+
+}
+
+.burger__line_first {
+	top: 0;
+}
+
+.burger__line_second {
+	top: 50%;
+	transform: translateY(-50%);
+}
+
+.burger__line_third {
+	bottom: 0;
+}
+/*header styles end*/
+
+
+/*content styles start*/
+
+.content {
+	display: flex;
+	margin-left: 10%;
+}
+
+.info {
+	flex: 2.5; /*ширина флекс элемента*/
+	justify-content: center; /*выравнивание элементов по ширине - центр*/
+	display: flex;
+	flex-flow: column; /*flex-элементы в виде блоков друг над другом*/
+	margin-right: 20%;
+}
+
 .info__title {
-	font-family: PTSerif
+	max-height: 400px; 
+	max-width: 530px; 
+}
+
+.info__title_big {
+	height: auto;
+	width: 527px;
+}
+
+.info__title_big img {
+	max-width: 100%;
+	max-height: 100%;
+}
+
+.info__title_small {
+	font-family: 'PT Serif'; 
+	font-style: normal;
+	font-size: 98px;
+	line-height: 130px;
+	margin: -70px 0 25px 70px; /*отступы, чтобы надпись съехала впрво вверх*/
+}
+
+.button__book {
+	width: 293px;
+	height: 62px;
+	background: #D4BEA3; /*цвет бэкграунда*/
+	border-radius: 48px; /*закругление кнопки*/
+	font: normal 22px/26px MuseoSansCyrl; /*стиль шрифта*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project-Bali/pullRequest?prid=5455738)